adapters/handlers/rest/clusterapi/indices_replicas.go

```diff
@@ -288,7 +286,7 @@ func (i *replicatedIndices) postObject() http.Handler {
 		switch ct {
 
 		case IndicesPayloads.SingleObject.MIME():
-			i.postObjectSingle(w, r, index, shard, requestID)
+			i.postObjectSingle(w, r, index, shard, requestID, schemaVersion)
 			return
 		case IndicesPayloads.ObjectList.MIME():
 			i.postObjectBatch(w, r, index, shard, requestID, schemaVersion)
```

Comment:
**question (non-blocking):** do we wanna do it also for batch ?

Reply:
You can see just below we were already doing it for batch. I had to add it for single object operations in this PR.

Reply:
sorry i miss read, didn't see it :/